tests/test_samples.py

```diff
@@ -913,3 +913,14 @@ def test_copy():
     assert new is not pc
     assert new.tex is not pc.tex
     assert new.limits is not pc.limits
+
+
+def test_plotting_with_integer_names():
+    np.random.seed(3)
+    samples_1 = MCMCSamples(data=np.random.rand(1000, 3))
+    samples_2 = MCMCSamples(data=np.random.rand(1000, 3))
+    fig, ax = samples_1.plot_2d([0, 1, 2])
+    samples_2.plot_2d(ax)
+
+    fig, ax = samples_1.plot_1d([0, 1, 2])
+    samples_2.plot_1d(ax)
```

Comment:
Might be good to expand this test beyond plotting to also test accessing integer labels. Would we have to write `samples_1['0']`? Would `samples_1.0` be possible?

Reply:
`samples_1.0` is not possible -- a standard pandas restriction sadly. Have added some tests to show how accessing works i.e. `samples_1[0]` is the right syntax, not `samples_1['0']`, although this is really also 'standard' pandas (by which I mean, a gotcha that seems to get me every time).